src/web/nextui/src/app/components/InfoModal.tsx

```diff
@@ -0,0 +1,65 @@
+import BugReportIcon from '@mui/icons-material/BugReport';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import ForumIcon from '@mui/icons-material/Forum';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { Box, Modal, Typography } from '@mui/material';
+import Link from 'next/link';
+import './InfoModal.css';
+
+export function InfoModal({ open, onClose }: { open: boolean; onClose: () => void }) {
+  return (
+    <Modal open={open} onClose={onClose} className="info-modal">
+      <Box className="info-modal-content">
+        <Typography component="h2" variant="h5" className="info-modal-title">
+          About Promptfoo
+        </Typography>
+        <Typography component="p" variant="body2" className="info-modal-description">
+          Promptfoo is a MIT licensed open-source tool for evaluating LLMs. We make it easy to track
+          the performance of your models and prompts over time with automated support for dataset
+          generation and grading.
+        </Typography>
+        <Typography component="h5" variant="h6" sx={{ marginTop: '.2em' }}>
+          <Link
```

Comment:
Put these in a Stack and you can get rid of the flexbox css https://mui.com/material-ui/react-stack/